docs/keycodes.md

```diff
@@ -0,0 +1,299 @@
+|`KC.WWW_HOME`          |`KC.WHOM`           |Browser Home (Windows)                         |
+|`KC.WWW_BACK`          |`KC.WBAK`           |Browser Back (Windows)                         |
+|`KC.WWW_FORWARD`       |`KC.WFWD`           |Browser Forward (Windows)                      |
+|`KC.WWW_STOP`          |`KC.WSTP`           |Browser Stop (Windows)                         |
+|`KC.WWW_REFRESH`       |`KC.WREF`           |Browser Refresh (Windows)                      |
+|`KC.WWW_FAVORITES`     |`KC.WFAV`           |Browser Favorites (Windows)                    |
+|`KC.MEDIA_FAST_FORWARD`|`KC.MFFD`           |Next Track (macOS)                             |
+|`KC.MEDIA_REWIND`      |`KC.MRWD`           |Previous Track (macOS)                         |
+
+
+## [US ANSI Shifted Symbols]
+
+|Key                     |Aliases            |Description        |
+|------------------------|-------------------|-------------------|
+|`KC.TILDE`              |`KC.TILD`          |`~`                |
+|`KC.EXCLAIM`            |`KC.EXLM`          |`!`                |
+|`KC.AT`                 |                   |`@`                |
+|`KC.HASH`               |                   |`#`                |
+|`KC.DOLLAR`             |`KC.DLR`           |`$`                |
+|`KC.PERCENT`            |`KC.PERC`          |`%`                |
+|`KC.CIRCUMFLEX`         |`KC.CIRC`          |`^`                |
+|`KC.AMPERSAND`          |`KC.AMPR`          |`&`                |
+|`KC.ASTERISK`           |`KC.ASTR`          |`*`                |
+|`KC.LEFT_PAREN`         |`KC.LPRN`          |`(`                |
+|`KC.RIGHT_PAREN`        |`KC.RPRN`          |`)`                |
+|`KC.UNDERSCORE`         |`KC.UNDS`          |`_`                |
+|`KC.PLUS`               |                   |`+`                |
+|`KC.LEFT_CURLY_BRACE`   |`KC.LCBR`          |`{`                |
+|`KC.RIGHT_CURLY_BRACE`  |`KC.RCBR`          |`}`                |
+|`KC.PIPE`               |                   |<code>&#124;</code>|
+|`KC.COLON`              |`KC.COLN`          |`:`                |
+|`KC.DOUBLE_QUOTE`       |`KC.DQUO`, `KC.DQT`|`"`                |
+|`KC.LEFT_ANGLE_BRACKET` |`KC.LABK`, `KC.LT` |`<`                |
+|`KC.RIGHT_ANGLE_BRACKET`|`KC.RABK`, `KC.GT` |`>`                |
+|`KC.QUESTION`           |`KC.QUES`          |`?`                |
+
+
+## [Internal Keycodes]
+
+|Key                    |Description                                                          |
+|-----------------------|---------------------------------------------------------------------|
+|`KC.RESET`             |Put the keyboard into DFU mode for flashing                          |
+|`KC.DEBUG`             |Toggle debug mode                                                    |
+|`KC.GESC`              |Escape when tapped, <code>&#96;</code> when pressed with Shift or GUI|
+|`KC.LEAD`              |The [Leader key]                                                     |
```

Comment:
This link goes nowhere

Reply:
We don't have the server set up, so I don't know where to send it yet.

Reply:
ahhh, fair